config/locales/en.yml

```diff
@@ -311,6 +311,120 @@ en:
+        to acquire the desired knowledge and skills on a subject by the end of the pathway.
+      editorial: >
+        <p>There are several roles associated with learning paths in general:</p>
+        <ul>
+        <li><b>Author</b>: Individuals and organisations involved in the preparation, creation or presentation 
+        of the published work.</li>
+        <li><b>Contributor</b>: Individuals and organisations that contributed to the published work but are not authors.</li>
+        <li><b>Collaborator</b>: An individual who has been granted editing rights to a material, topic or learning path 
+        by the individual (owner) who created it.</li>
+        <li><b>Curator</b>: An individual who has been granted permission to create a learning path in TeSS
+        while following their community&rsquo;s editorial process.</li>
+        <li><b>Owner</b>: An individual who has created a material, topic or learning path in TeSS.</li></ul>
+        <p>Before registering a learning path, a training provider must: </p>
+        <ol><li>Assign a learning path curator (owner).</li>
+        <li>Contact the TeSS team to assign the learning path curator role to their account.</li></ol>
+        <p>After these tasks are completed:</p>
+        <ul><li>A learning path curator has permission to create learning paths for their provider.</li>
+        <li>It is the responsibility of the learning path curator to curate and implement any protocols 
+        required by the community providing the learning path. 
+        For example, learning paths from the ELIXIR community will follow the protocol created by the 
+        ELIXIR Training Platform/Learning Paths Focus Group.</li></ul>
+        <p>Governance: Maintaining the learning path</p>
+        <ol><li>It is the responsibility of the learning path curator to maintain their community&raquo;s learning paths.</li>
+        <li>The curator can assign collaborators to work on each learning path, depending on the expertise required.</li></ol>
+      overview: >
+        <p>To register a learning path you need to follow the three steps described below:</p>
+        <ol><li>Register Training Materials</li>
+        <li>Create a learning path topic and add materials to it (repeat for each topic)</li>
+        <li>Register a learning path and add learning path topics to it</li></ol>
+        <p>To register a learning path and its topics, first log in. The 'Register learning path' button 
+        will become visible on the learning paths page, and the 'Create topic' button will become visible 
+        on the learning paths topic page. If you do not see these buttons, contact 
+        tess-support@googlegroups.com to request permission.</p>
+        <p>The three steps to register a learning path are described in full detail below.</p> 
+      training_material: >
+        <p>To <a href='about/registering'>register training materials follow the instructions here</a>. 
```

Comment:
For the link here its better to let Rails generate the correct path. You can do it like we did here:
https://github.com/ElixirTeSS/TeSS/blob/1e2efc7b3dfe9f88a20403fc1da765a62b63f59b/app/views/about/tess.html.erb#L105

in this case it would be `registering_resources_path`